.swiftlint.yml

```diff
@@ -0,0 +1,16 @@
+included:
+  - Sources
+  - Tests
+excluded:
+  - Tests/**/*Resources # files in this directory contains only JSON strings.
+  - Tests/**/XCTestManifests.swift # auto-generated file
+line_length:
+  warning: 140 # needs discussion (default: 120)
+  error: 350 # needs discussion (default: 200)
+  ignores_comments: true
+  ignores_urls: true
+  ignores_function_declarations: true
+  ignores_interpolated_strings: true
+identifier_name:
+  min_length: 
+    warning: 2
```

Comment:
Can we use `excluded:` to add the excluded words instead? (like id)

Reply:
Sure! fixed in 38131f0